src/device/disableScroll.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Another way to do this would be to have a single default export that takes a boolean argument, e.g. `allowScrolling(false)`.

Reply:
I went ahead and refactored this since it took minimal effort.